src/SourceCode.Chasm/TreeNode.cs

```diff
@@ -23,22 +23,6 @@ public struct TreeNode : IEquatable<TreeNode>, IComparable<TreeNode>
         /// </value>
         public static TreeNode Empty { get; }
 
-        /// <summary>
```

Comment:
Why?

Reply:
The SHA node is not nullable - a null treenode is simply missing from the parent.

Reply:
ok